addons/sale/views/sale_order_line_views.xml

```diff
@@ -68,15 +68,21 @@
                             <field name="tax_country_id" invisible="1"/>
                             <field name="price_unit"/>
                             <field name="discount" groups="sale.group_discount_per_so_line"/>
-                            <field name="price_subtotal" widget="monetary"/>
+                            <field name="price_subtotal"
+                                   widget="monetary"
+                                   invisible="company_id.company_price_include == 'tax_included'"
```

Comment:
Is it working? I think you should use your related field instead

Reply:
It's on the order though, not on the line. I'm checkin'.

Reply:
It's actually working.
![immagine](https://github.com/user-attachments/assets/c86a9270-d81c-4293-a817-dde2b7f8e6f7)
![immagine](https://github.com/user-attachments/assets/84276fc8-b14c-4b90-a7e3-96401752e41a)

Reply:
Now that I think again about it, you miss the part where @chklop said that showing one column (tax_included, tax_excluded) or another should depend on the company's default, not on the type of any taxes being shown.

Reply:
The screenshots are not on this view, but on `view_order_form` which correctly uses `parent.company_price_include`.
In your case, you will get `undefined`. (`parent` is a special variable, the only one allowing to do "relational" stuff)